neurolib/models/multimodel/builder/base/network.py

```diff
@@ -1,14 +1,37 @@
 import logging
+from copy import deepcopy
 from itertools import chain, islice
 
 import numpy as np
 import symengine as se
+import sympy as sp
 from jitcdde import t as time_vector
 from jitcdde import y as state_vector
 
+from .....utils.collections import flat_dict_to_nested, flatten_nested_dict
 from .backend import BackendIntegrator
 from .constants import EXC, NETWORK_CONNECTIVITY, NETWORK_DELAYS, NODE_CONNECTIVITY, NODE_DELAYS
 from .neural_mass import NeuralMass
+from .params import float_params_to_individual_symbolic, float_params_to_vector_symbolic
+
+
+def _sanitize_matrix(matrix, target_shape):
```

Comment:
when connectivity matrix elements are integers and not floats, for some reason we have a bug... this asserts float connectivity elements